### PR TITLE
feat: create ThirdPartyClaimsCertificate & supporting schemas

### PR DESCRIPTION
**Pull request by rhofvendahl**
This PR creates a certificate for claims made by a third party (as requested by @mprorock ).

### CODE REVIEW

File: docs/openapi/components/schemas/common/Claim.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      term: claim
+      '@id': https://schema.org/name
+  value:
+    title: Value
+    description: The value of a claim.
+    type: string
+    $linkedData:
+      term: value
+      '@id': https://schema.org/value
+  valueObject:
```
Comment (mkhraisha) [inline docs/openapi/components/schemas/common/Claim.yml:31]: is this the same as Value just as an object not a string?

**Review by mkhraisha — APPROVED** [review]

File: docs/openapi/components/schemas/common/Claim.yml (hunk 1 above)
Comment (rhofvendahl) [reply docs/openapi/components/schemas/common/Claim.yml:31]: Yeah, Mike said values would be either strings or objects & this seemed like a reasonable approach.

File: docs/openapi/components/schemas/credentials/ThirdPartyClaimsCertificate.yml [hunk 2]
```diff
@@ -0,0 +1,81 @@
+$linkedData:
+  term: ThirdPartyClaimsCertificate
+  '@id': https://w3id.org/traceability#ThirdPartyClaimsCertificate
+title: Third Party Claims Certificate
```
Comment (BenjaminMoe) [inline docs/openapi/components/schemas/credentials/ThirdPartyClaimsCertificate.yml:4]: Add tag?

**Comment by nissimsan** [comment]
@mprorock, we discussed this a little on the call. The intention of this PR confuses me. This looks like generic claims, but isn't that what a VerifiableCredential already is? 
What am I missing?

**Review by mprorock — APPROVED** [review]

**Comment by mprorock** [comment]
> @mprorock, we discussed this a little on the call. The intention of this PR confuses me. This looks like generic claims, but isn't that what a VerifiableCredential already is? What am I missing?

It totally is.  This is based on a request fr two different customers to provide a formal schema for description of a link between a system external to trace and a traceable presentation in a repeatable and flexible manner so that it facilitates an easier upgrade path with opaque customer systems

**Comment by TallTed** [comment]
This feels to me to be essentially a new costume for a Verifiable Presentation (VP)... 

Most of the use case appears to be business logic to handle such a VP, which we *could* standardize, but I wonder whether that won't effectively impose limitations that diminish the applicability and utility of VPs for entities that are not currently participating in this standardization process.

(I also wonder why this began as a PR, and not an Issue.)

**Review by OR13 — CHANGES_REQUESTED** [review]
This seems like an attempt to reinvent JSON-LD on JSON-LD.

**Comment by nissimsan** [comment]
Re-inventing https://www.w3.org/TR/vc-data-model/#claims ?

**Comment by nissimsan** [comment]
General pushback on the trace call. Suggestion to bring this up on an issue first.

**Comment by OR13** [comment]
We also discussed how https://www.w3.org/TR/vc-data-model/#evidence might be a useful way to represent this.

**Comment by rhofvendahl** [comment]
@brownoxford Yes, doing that now.

**Pull request closed without merging**